Extension/src/LanguageServer/client.ts

```diff
@@ -1861,6 +1863,81 @@ export class DefaultClient implements Client {
         await this.notifyWhenLanguageClientReady(() => this.languageClient.sendNotification(RescanFolderNotification));
     }
 
+    public async addIncludeHeader(filePath: string, range: vscode.Range, token: vscode.CancellationToken): Promise<void> {
```

Comment:
client.ts is already huge. I will extract the function if the feature is approved by a codeowner